6-7-8/script.js

```diff
@@ -0,0 +1,84 @@
+"use strict"
+
+let tasks;
+!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks'));
+
+let todoItemElems = [];
+
+class createTask {
+    constructor(about) {
+        this.about = about;
+        this.datetask = creationDate();
+    }
+}
+
+const createTemplate = (task, event) => {
```

Comment:
это же индекс элемента в массиве, это же не event
```suggestion
const createTemplate = (task, index) => {
```